Blogplace.Tests.Integration/Tests/CommentsTests.cs

```diff
@@ -0,0 +1,171 @@
+﻿using Blogplace.Tests.Integration.Data;
+using Blogplace.Web.Domain.Comments;
+using Blogplace.Web.Domain.Comments.Requests;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc.Testing;
+using System.Net;
+using System.Net.Http.Json;
+
+namespace Blogplace.Tests.Integration.Tests;
+
+public class CommentsTests : TestBase
+{
+    private WebApplicationFactory<Program> _factory;
+
+    [SetUp]
+    public void SetUp() => this._factory = StartServer();
+
+    [TearDown]
+    public void TearDown() => this._factory?.Dispose();
+
+    [Test]
+    public async Task Comments_AnonymousShouldReturnUnauthorized()
+    {
+        //Arrange
+        var client = this._factory.CreateClient_Anonymous();
+        var requests = new Dictionary<string, object>();
+        var createRequest = new CreateCommentRequest(
+            ArticlesRepositoryFake.StandardUserArticle!.Id,
+            "TEST_COMMENT_CONTENT"
+        );
+        var updateRequest = new UpdateCommentRequest(
+            CommentsRepositoryFake.StandardUserCommentOnStandardUserArticle!.Id,
+            "NEW_CONTENT"
+        );
+        var deleteRequest =
+            new DeleteCommentRequest(CommentsRepositoryFake.StandardUserCommentOnStandardUserArticle!.Id);
+
+        requests["/Comments/Create"] = createRequest;
+        requests["/Comments/Update"] = updateRequest;
+        requests["/Comments/Delete"] = deleteRequest;
+
+        foreach (var request in requests)
+        {
+            //Act
+            var response = await client.PostAsync($"{this.urlBaseV1}{request.Key}", request.Value);
+
+            //Assert
+            response.StatusCode.Should().Be(HttpStatusCode.Unauthorized);
+        }
+    }
+
+    [Test]
+    public async Task Create_CommentShouldBeCreated()
+    {
+        //Arrange
+        var client = this._factory.CreateClient_Standard();
+        var request = new CreateCommentRequest(
+            ArticlesRepositoryFake.StandardUserArticle!.Id,
+            "TEST_COMMENT_CONTENT"
+        );
+
+        //Act
+        var response = await client.PostAsync($"{this.urlBaseV1}/Comments/Create", request);
+
+        //Assert
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+        // TODO: check if the comment actually exist in the repository
```

Comment:
łatwo można wyciągnąć repo z `_factory`, ale można zrobić to requestem i od razu upewniać sie czy zwraca dodane komentarze